tsdef/RemoteAudioTrack.d.ts

```diff
@@ -5,14 +5,24 @@ export class RemoteAudioTrack extends AudioTrack {
   sid: Track.SID;
   priority: Track.Priority | null;
   isSwitchedOff: boolean;
+  switchOffReason: Track.SwitchOffReason | null;
+  /**
+   * @deprecated Use (.isSwitchedOff && .switchOffReason === Track.SwitchOffReason.DISABLED_BY_PUBLISHER) instead
+   */
```

Comment:
Should this be:
```suggestion
  /**
   * @deprecated Use (.switchOffReason !== Track.SwitchOffReason.DISABLED_BY_PUBLISHER) instead
   */
```
(ditto for RemoteVideoTrack)